labelbox/schema/organization.py

```diff
@@ -4,6 +4,7 @@
 from labelbox import utils
 from labelbox.orm.db_object import DbObject, query, Entity
 from labelbox.orm.model import Field, Relationship
+from labelbox.schema.invite import InviteLimit
 
 if TYPE_CHECKING:
     from labelbox import Role, User, ProjectRole, Invite, InviteLimit, IAMIntegration
```

Comment:
Why not Entity.class name? Do you need to merge the type checking pr in here?

Reply:
yeah InviteLimit is not an Entity but everything else is. So just remove all the other imports here

Reply:
I guess a role isn't either.

Reply:
Just make sure all Entities are always type checked with Entity.Name. instead of if TYPE_CHECKING.